app/controllers/sign_up/registrations_controller.rb

```diff
@@ -11,16 +11,21 @@ class RegistrationsController < ApplicationController
     CREATE_ACCOUNT = 'create_account'
 
     def new
-      @register_user_email_form = RegisterUserEmailForm.new(analytics:)
+      @register_user_email_form = RegisterUserEmailForm.new(analytics:, attempts_api_tracker:)
       analytics.user_registration_enter_email_visit
     end
 
     def create
-      @register_user_email_form = RegisterUserEmailForm.new(analytics:)
+      @register_user_email_form = RegisterUserEmailForm.new(analytics:, attempts_api_tracker:)
 
       result = @register_user_email_form.submit(permitted_params.merge(request_id:))
 
       analytics.user_registration_email(**result)
+      attempts_api_tracker.user_registration_email_submitted(
+        email: permitted_params[:email] || '',
```

Comment:
Is the conditional empty string needed?

Reply:
so, needed is a strong word, but i did this to address an inconsistency i saw when i was running the tests. 

if the email params is nil (`{ email: nil }`) then `permitted_params[:email]` evaluates to an empty string.
if the params is a hash (`{email:  { foo: 'bar' }}`) then `permitted_params[:email]` evaluates to nil. 

i thought the inconsistency was bad, although don't have an opinion whether `nil` or `''` is a better default. (i don't know if the params being a hash is actually a concern, but the test was there, so i took it seriously)

Reply:
updated in https://github.com/18F/identity-idp/pull/12175/commits/d2b99ec7b16b5acf040f96f275459d2392d565a3